portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,68 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Displays random fact about me
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  var num = 100;
+  const facts =
+      ['I am 20 years old', 'I live in Colorado', 'I am a human', 'I am not a robot'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factsContainer = document.getElementById('fact');
+  factsContainer.innerText = fact;
+
+  // Adds function to move text back and forth
+  if (factsContainer.style.marginLeft == "")
+    setInterval( function() {
+            factsContainer.style.marginLeft = (Math.abs(100 - num)%100).toString() + "px";
+            num = num % 199 + 1;
+            console.log((Math.abs(100 - num)%100).toString() + "px");
+        }, 20);
+}
+
+function hideUnhide(content)  {
```

Comment:
A more canonical way to do this is to have a hidden class in CSS that styles any element on it with display none:

.hidden {
  display: none;
}

Then you toggle whether the class is added to or removed from the element, without worrying about the style element.

Reply:
That's done.